Composer/packages/types/src/runtime.ts

```diff
@@ -37,11 +37,15 @@ export type BotTemplate = {
   name: string;
   description: string;
   /* absolute path */
-  path: string;
+  path?: string;
   /* tags for further grouping and search secenario */
   tags?: string[];
   /* list of supported runtime versions */
   support?: string[];
+  packageName?: string;
```

Comment:
I'm wondering if you can do:
```
package?: {
  name: string;
  source: string;
  version: string;
}
```

This way when there is a package, there must be a name, version and source.